monitoring/configuration/ks-transport-ctl-svc.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: ks-transport-controller
+  name: ks-transport-controller
+spec:
+  ports:
+  - name: http1
```

Comment:
Do you anticipate having a port named "http2"? I do not think that "1" vs "2" conveys any useful information. I suggest more informative names.

Reply:
Fixed. Add a better name: `http-metrics`